x-pack/test/functional/apps/ml/short_tests/model_management/model_list.ts

```diff
@@ -144,6 +146,21 @@ export default function ({ getService }: FtrProviderContext) {
           { pipelineName: `pipeline_${modelWithPipelineData.modelId}`, expectDefinition: false },
         ]);
       });
+
+      it('the add trained model flyout should display elements on Manual Download tab correctly', async () => {
+        await ml.testExecution.logTestStep('Open the Add Trained Model Flyout');
+        await ml.trainedModelsFlyout.assertOpen(true);
```

Comment:
As we discussed in Slack, this method should be called `open` and make assertion internally. 

```suggestion
        await ml.trainedModelsFlyout.open(true);
```